chart/helm-operator/templates/rbac.yaml

```diff
@@ -20,8 +19,6 @@ rules:
       - '*'
     verbs:
       - '*'
-{{- end -}}
-{{- if or .Values.clusterRole.create .Values.clusterRole.name }}
```

Comment:
The different conditional here means that using

    --set clusterRole.create=false --set clusterRole.name=foobar

will have a different result to before. I think this is OK as a change, as mentioned in the comments section; please update the description of these settings in `values.yaml` so it's clear that `clusterRole.create: false` will now suppress the ClusterRole _and_ ClusterRoleBinding.

Reply:
gotcha, will make that change in the next few minutes.

Reply:
Done.